containers.go

```diff
@@ -368,16 +381,61 @@ func (r *containerStore) BigData(id, key string) ([]byte, error) {
 }
 
 func (r *containerStore) BigDataSize(id, key string) (int64, error) {
+	if key == "" {
+		return -1, errors.Wrapf(ErrInvalidBigDataName, "data name %q can not be used as a filename", key)
```

Comment:
Ok, after going through the whole review , I see this error in a number of places.  I think it's misleading to tell them "data name  can not be used as a file name."  Perhaps "data key was empty and is invalid for a filename" or something along those lines?

Reply:
Well, I want to keep the detail that ultimately this is used to construct a filename private, so that we don't implicitly promise that it'll always stay that way.  But I agree that we could be more specific when rejecting an empty string, since the error code itself already says that it's a bad name in general, which made the detail the wrapping step was adding not very helpful.